results-tabulation-api/readme.md

```diff
@@ -12,6 +12,9 @@
 
```

Comment:
Shall we make this also two sections as above

Reply:
sure, will do